database/seeding.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs');
+const path = require('path');
+
+function getRandomName() {
```

Comment:
if you want you can rewrite this to grab a random item from a list by using `firstName.length` rather than hard setting 300